### PR TITLE
Add try and claim CLI commands

### PR DESCRIPTION
**Pull request by michaeladda**
manually tested with matching binaris/spice#644

### CODE REVIEW

File: cli/src/commands/list.ts [hunk 1]
```diff
@@ -20,8 +20,8 @@ export default class List extends Command {
       return;
     }
     // TODO: Get URL from lycan
```
Comment (bergundy) [inline cli/src/commands/list.ts]: Remove the TODO please.

File: cli/src/commands/claim.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      name: 'token',
+      required: true,
+      description: 'claim token',
+    },
+  ];
+
+  public static strict = true;
+
+  public async run() {
+    const { args } = this.parse(Claim);
```
Comment (bergundy) [inline cli/src/commands/claim.ts]: You should check that you're authenticated here: https://github.com/binaris/shiftjs/blob/b76a1672cde2422afa48e346043584ab9f766ac9/cli/src/commands/deploy.ts#L116

File: cli/src/commands/list.ts (hunk 1 above)
Comment (bergundy) [reply cli/src/commands/list.ts]: You should authenticate() above as well.

**Comment by michaeladda** [comment]
Thanks for review. Addressed issues and found a few additional similar problems so fixed those too. @bergundy PTAL :-)

**Comment by michaeladda** [comment]
Excellent. Waiting for binaris/spice#644 before merging

**Pull request merged**